spec/unit/translators/nsidc_json_to_solr_spec.rb

```diff
@@ -230,50 +230,26 @@
     end
   end
 
-  describe 'featured faceting' do
-    it 'translates true cumulus value to featured facet value' do
+  describe 'storage location faceting' do
+    it 'translates true cumulus value to storage location facet value' do
       json_doc = { 'cumulus' => true }
-      facet = translator.translate_featured(json_doc, nil)
+      facet = translator.translate_storage_location(json_doc)
       expect(facet).to include('In Earthdata Cloud')
       expect(facet).not_to include('Show Global Only')
     end
 
-    it 'does not translate false cumulus value to featured facet value' do
+    it 'does not translate false cumulus value to storage location facet value' do
       json_doc = { 'cumulus' => false }
-      facet = translator.translate_featured(json_doc, nil)
+      facet = translator.translate_storage_location(json_doc)
       expect(facet).not_to include('In Earthdata Cloud')
       expect(facet).not_to include('Show Global Only')
     end
 
-    it 'does not translate missing cumulus value to featured facet value' do
+    it 'does not translate missing cumulus value to storage location facet value' do
       json_doc = {}
-      facet = translator.translate_featured(json_doc, nil)
+      facet = translator.translate_storage_location(json_doc)
       expect(facet).not_to include('In Earthdata Cloud')
       expect(facet).not_to include('Show Global Only')
     end
-
-    it 'translates global spatial coverage to featured facet value' do
```

Comment:
Any reason to not include (any of) these tests for the "Spatial Coverage" section?

Reply:
I don't think we need to.  There are already tests in place that test the "geojson_to_global_facet" called at line 59, and since the facet value is being set directly to that, I don't think there's a need for more tests here.

The reason I had these here before was because these tests were largely testing the "facet_featured" facet itself, ensuring that the Cloud and Global options appeared (or not) in that facet.  I think testing the "facet_spatial_coverage" facet would really just be testing "geojson_to_global_facet" again.